### PR TITLE
Update Docker installation instructions

### PR DESCRIPTION
**Pull request by nilsvu**
... and add Python+Jupyter instructions.

### Component:

- [ ] Code
- [x] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests and `clang-tidy`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: docs/Tutorials/Python.md [hunk 1]
```diff
@@ -24,3 +24,41 @@ supports `pip`, for instance in a
...
+
+## Running Jupyter within the Docker container
+
+[Jupyter lab](https://jupyterlab.readthedocs.io/) is installed in the Docker
+container. You can run it in the container and access it through a browser on
+the host for a convenient way to work with the SpECTRE Python bindings. To do
+so, make sure you have exposed a port when running the Docker container, e.g.
+by appending the option `-p 8000:8000` to the `docker run` command (see
+\ref installation). Inside the docker container, it can be convenient to
+`apt-get install screen` and use `screen` to obtain a shell that runs the
```
Comment (nilsdeppe) [inline docs/Tutorials/Python.md]: Why prefer screen over disowning the job?

File: docs/Tutorials/Python.md [hunk 2]
```diff
@@ -24,3 +24,41 @@ supports `pip`, for instance in a
 access to the SpECTRE Python modules by adding  `BUILD_DIR/bin/python` to your
 `PYTHONPATH`. This is done automatically by sourcing the `LoadPython.sh` file
 with the command `. BUILD_DIR/bin/LoadPython.sh`.
+
```
Comment (nilsdeppe) [inline docs/Tutorials/Python.md:27]: Could you add this file to the `Tutorials` page? Currently if I go to `User Tutorials` it doesn't show up, but given its location I think this was an oversight when it was added.

File: docs/Installation/Installation.md [hunk 3]
```diff
@@ -85,16 +85,25 @@ To build with the docker image:
...
+   bindings (see \ref spectre_using_python) or a Python web server to view the
+   documentation. To do so, append the `-p` option, e.g. `-p 8000:8000`.
+
+   You will end up in a shell in the docker container,
    as root (you need to be root).
-   Within the container, SPECTRE_ROOT is available and
-   CHARM_DIR is /work/charm. For the following steps, stay inside the docker
-   container as root.
+   Within the container, the files in SPECTRE_ROOT are available and
+   Charm++ is installed in /work/charm. For the following steps, stay inside the
```
Comment (nilsdeppe) [inline docs/Installation/Installation.md]: Do you want to put ticks around `/work/charm` for consistency?

**Review by nilsdeppe — CHANGES_REQUESTED** [review]

File: docs/Tutorials/Python.md (hunk 1 above)
Comment (nilsvu) [reply docs/Tutorials/Python.md]: No reason, only a suggestion. Do you prefer to remove this sentence?

Comment (nilsdeppe) [reply docs/Tutorials/Python.md]: No, seems useful. Maybe add disowning as an alternative screen? screen can be a bit awkward to use when you first get started.

Comment (nilsvu) [reply docs/Tutorials/Python.md]: ok I'm namedropping `disown` now so people can look it up when they are interested (same for `screen`)

**Comment by nilsdeppe** [comment]
Looks good! Go ahead and squash :)

File: docs/Tutorials/Python.md [hunk 4]
```diff
@@ -24,3 +24,41 @@ supports `pip`, for instance in a
...
+## Running Jupyter within the Docker container
+
+[Jupyter lab](https://jupyterlab.readthedocs.io/) is installed in the Docker
+container. You can run it in the container and access it through a browser on
+the host for a convenient way to work with the SpECTRE Python bindings. To do
+so, make sure you have exposed a port when running the Docker container, e.g.
+by appending the option `-p 8000:8000` to the `docker run` command (see
+\ref installation). Inside the docker container, it can be convenient to
+use `disown` or to `apt-get install screen` and use `screen` to obtain a shell
```
Comment (fmahebert) [inline docs/Tutorials/Python.md:36]: Suggestion: "... use `disown` or `screen` (install this inside the container with `apt-get install screen`) to obtain..."

File: docs/Tutorials/Python.md [hunk 5]
```diff
@@ -24,3 +24,41 @@ supports `pip`, for instance in a
...
+that runs the Jupyter server permanently in the background. Within the shell you
+want to run your Jupter server, navigate to a directory that will serve as the
```
Comment (fmahebert) [inline docs/Tutorials/Python.md]: typo: Jupter

**Review by fmahebert — COMMENTED** [review]
I haven't tried the macOS instructions, but overall this is readable and (to me) clear. 👍

Two very minor wording items (squash any changes immediately).

File: docs/Tutorials/Python.md (hunk 4 above)
Comment (nilsvu) [reply docs/Tutorials/Python.md:36]: `disown` doesn't give you a new shell, so that sounds misleading I think?

Comment (fmahebert) [reply docs/Tutorials/Python.md:36]: That's a fair point, so let's forget this suggestion.

File: docs/Tutorials/Python.md (hunk 5 above)
Comment (nilsvu) [reply docs/Tutorials/Python.md]: fixed

**Comment by kidder** [comment]
@fmahebert @nilsdeppe please look at

**Review by fmahebert — APPROVED** [review]

**Review by nilsdeppe — APPROVED** [review]

**Comment by kidder** [comment]
ignoring travis timeout

**Pull request merged**